src/NLog.Targets.Syslog/MessageCreation/Rfc5424.cs

```diff
@@ -57,7 +57,7 @@ protected override void PrepareMessage(ByteArray buffer, LogEventInfo logEvent,
 
         private void AppendHeader(ByteArray buffer, string pri, LogEventInfo logEvent)
         {
-            var timestamp = string.Format(CultureInfo.InvariantCulture, TimestampFormat, logEvent.TimeStamp);
+            var timestamp = timestampLayout.Render(logEvent);

```

Comment:
If you try running one of the test apps you will see that this is not equivalent to the line it replaces